src/QueueOperation.h

```diff
@@ -46,7 +46,7 @@ class QueueOperation {
 	enum QueueType { QueueTypeConnect, QueueTypeDisconnect, QueueTypeDownload, QueueTypeUpload,
 	                 QueueTypeDirectoryGet, QueueTypeDirectoryCreate, QueueTypeDirectoryRemove,
 	                 QueueTypeFileCreate, QueueTypeFileDelete, QueueTypeFileRename, QueueTypeQuote,
-	                 QueueTypeDownloadHandle
+	                 QueueTypeDownloadHandle,QueueTypeCopyFile

```

Comment:
Add a space to be consistent with the formatting of the other types.

Reply:
ok